nuget.config

```diff
@@ -1,8 +1,9 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
     <clear />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <add key="Toolkit Labs" value="https://pkgs.dev.azure.com/dotnet/CommunityToolkit/_packaging/CommunityToolkit-Labs/nuget/v3/index.json" />
```

Comment:
This nuget.config file will be replaced with the one in our internal repo for the release pipeline. Let me handle this in our internal repo at first and then we can merge this change to the GitHub repo. Otherwise, the settings page will get broken.

Reply:
Done.